meson_options.txt

```diff
@@ -8,7 +8,8 @@ option('enable-uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('enable-uefi-labels', type : 'boolean', value : true, description : 'enable UEFI labels support')
 option('enable-dell', type : 'boolean', value : true, description : 'enable Dell-specific support')
 option('enable-amt', type : 'boolean', value : true, description : 'enable Intel AMT support')
-option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable Thunderbolt support')
+option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable legacy Thunderbolt support')
```

Comment:
In general we're trying to encourage distros to provide packaging in contrib/ in the project too.  At least for mainstream distros we should have a pretty good idea if they're going to deviate from project defaults.  Given there are not other explicit dependencies needed for thunderbolt I don't forsee many distros disabling this by default.